src/Login/Login.js

```diff
@@ -48,8 +50,38 @@ const Login = ({navigation}) => {
                 })
             }
         } else {
-            firebase.auth().signInWithEmailAndPassword(username, password).then(function(user) {
-                var user = firebase.auth().currentUser;
+            firebase.auth().setPersistence(firebase.auth.Auth.Persistence.LOCAL) // persistent login can only be cleared on user sign out
```

Comment:
firebase method to set persistent session
Current set as LOCAL _(other values are SESSION and NONE)_